src/renderer/components/recording-page/recording-page.test.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable no-console */
+/* eslint-disable jest/prefer-expect-assertions */
+
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import React from 'react';
+
+import { noOp } from '../../env-and-consts';
+import mediaService from '../../services/media';
+
+import { RecordingPage } from './index';
+
+jest.mock('../../services/media', () => ({
+  switchOnAudioInput: jest.fn(noOp),
+  stopRecording: jest.fn(noOp),
+  startRecording: jest.fn(noOp),
+  audioBlobToWavArrayBuffer: jest.fn(noOp),
+  createNewAudioGraph: jest.fn(noOp),
+  createOscillator: jest.fn(noOp),
+  playAudioNode: jest.fn(noOp),
+}));
+
+const originalError = console.error;
+
+const flushPromises = () => new Promise(setImmediate);
+const beforeTest = () => {
+  // Supresses compatibility issues with ReactDOM 16.8
+  console.error = (...args: string[]) => {
+    if (/Warning.*not wrapped in act/.test(args[0])) {
+      return;
+    }
+    originalError.call(console, ...args);
+  };
+};
+const afterTest = () => {
+  flushPromises();
+  console.error = originalError;
+  cleanup();
+};
+
+describe('recordingPage', () => {
+  it('starts recording when r is pressed', async () => {
+    beforeTest();
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (global as any).__static = '';
```

Comment:
I think `global as unknown` will also work

Reply:
I tried changing it to (global as unknown).__static = '';  and it gave me errors - do you want me to hunt down a fix that will allow me to use unknown, or should i just leave it as any?

Reply:
Show me the error that bugs `unknown`.
Also, look into setting the global variables for jest.

Reply:
![image](https://user-images.githubusercontent.com/16734869/111915737-26ebc580-8a4e-11eb-9d48-3f559795b2a1.png)
![image](https://user-images.githubusercontent.com/16734869/111915747-31a65a80-8a4e-11eb-8425-7f961aa19d19.png)

Reply:
Seems like it should still be set in jest config rather than in the test code because TypeScript doesn't like that.